README.md

```diff
@@ -450,8 +583,28 @@ Thank you to [Brendan O'Brien](https://github.com/b5) for real-world feedback, t
 
 Many thanks to [Irakli Gozalishvili](https://github.com/Gozala) for feedback and recommendations, contributing significantly to the TypeScript implementation, and suggesting renaming the action field to `can`.
 
+Thank you [Blaine Cook](https://github.com/blaine) for the real-world feedback, ideas on future features, and lessons from other auth standards.
+
 Thank you [Dan Finlay](https://github.com/danfinlay) for being sufficiently passionate about OCAP that we realized that capability systems had an actual chance of adoption in an ACL-dominated world.
 
 Thanks to the entire [SPKI WG](https://datatracker.ietf.org/wg/spki/about/) for their closely related pioneering work.
 
 We want to especially recognize [Mark Miller](https://github.com/erights) for his numerous contributions to the field of distributed auth, programming languages, and computer security writ large.
+
+# 9. FAQ
+
+## 9.1 What prevents an unauthorized party from using an intercepted UCAN?
+
+UCANs always contain the information about the sender and receiver. It is signed by the sender (the `iss` field DID), and so can only be created by an agent in posession of the relevant private key. The recipient (the `aud` field DID) is required to check that the field matches their DID. These two checks taken together secures against use by an unauthorized party.
```

Comment:
The second sentence can be strengthened a bit by repeating UCAN. Something like:

> A UCAN is signed by the sender (the `iss` field DID)

Typo in the last sentence:

> These two checks taken together ~~secures~~ secure against use by an unauthorized party.